sdk/file_sdk/Common/extended_property.h

```diff
@@ -2,14 +2,21 @@
 #define EXTENDED_PROPERTY_H_
 
 #include <string>
-
+#include "string_utils.h"
 namespace mip {
 
 struct ExtendedProperty
 {
   std::string vendor;
   std::string key;
   std::string value;
+
+  bool operator==(const ExtendedProperty& property) const
+  {
+    return EqualsIgnoreCase(key, property.key) &&
+           EqualsIgnoreCase(value, property.value) &&
+           EqualsIgnoreCase(vendor, property.vendor);

```

Comment:
are you sure that value should be case insensitive too?

Reply:
why not? it doesn't make any sense to add 2 properties with the same value but with different case

---
In reply to: [137451942](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r137451942) [](ancestors = 137451942)

Reply:
the case may be important for vendors

---
In reply to: [137476973](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r137476973) [](ancestors = 137476973,137451942)